### DISCUSSION TITLE
Перевод Unix timestamp в формат d.m.Y H:i

### DISCUSSION THREAD

**Discussion by an220022** [totumonline/totum-ru-issues-and-discussions#35] [Как это сделать?] [answered]
Есть ли какой-то простой способ перевода Unix timestamp в формат d.m.Y H:i?

**Comment by poluninatm** [comment, marked as answer]
При включении в Conf.php 
```
protected $execSSHOn = true;
```

Становится доступна функция execSSH. В линукс системе команда date может выполнять в ssh это преобразование (зависит от системы, в mac os у меня не работает, на сервере с Ubuntu - возвращает искомое.

Вот вариант кода действия кнопки, для указанного преобразования:

````
= : linkToDataText(title: "test"; text: $code)
code: execSSH(ssh: $replace)
replace: strReplace(str: "date +'%d.%m.%Y %H:%M' -d '@unixtime'"; from: 'unixtime'; to: 1637441119)
````

**Reply by ttm-alexeypolunin** [reply]
### Начиная с версии 2 все функции работы с датами в date принимают timestamp
